pratishtha_awas/settings.py

```diff
@@ -75,18 +82,19 @@
 
 # Database
 # https://docs.djangoproject.com/en/5.0/ref/settings/#databases
+
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': BASE_DIR / 'db.sqlite3',
     },
     'remote': {
-        'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'pratishtha',
-        'USER': os.environ['db_username'],
-        'PASSWORD': os.environ['db_password'],
-        'HOST': os.environ['db_ip'],
-        'PORT': os.environ['db_port'],

```

Comment:
This change is probably an error. The remote db configuration with mysql db should be the correct one.